fraud/bad_encoding.go

```diff
@@ -99,6 +99,8 @@ func (p *BadEncodingProof) UnmarshalBinary(data []byte) error {
 		headerHash:  in.HeaderHash,
 		BlockHeight: in.Height,
 		Shares:      ipld.ProtoToShare(in.Shares),
+		Index:       uint8(in.Index),
+		isRow:       in.IsRow,
```

Comment:
Is not this supposed to be part of findings Pr?

Reply:
wdym?

Reply:
Ah, got you. Befp won't work correctly without this 😅

Reply:
👍